src/components/ui/Tag/index.js

```diff
@@ -0,0 +1,147 @@
+import { css } from '@emotion/core';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+import React, { forwardRef } from 'react';
+
+import { Icon } from 'components/ui/Icon';
+import { getContrastingTextColor, metadata, toUnits } from 'styles';
+import { useTheme } from 'themes';
+
+export const Tag = forwardRef((props, ref) => {
+  const theme = useTheme();
+  const { children, label, onDismiss, ...otherProps } = props;
+
+  return (
+    <span ref={ref} {...otherProps}>
+      {label && (
+        <span // TODO: switch with VisuallyHidden component, once available.
+          css={css`
+            font-style: bold;
+            position: absolute;
+            left: -10000px;
+            top: auto;
+            width: 1px;
+            height: 1px;
+            overflow: hidden;
+          `}
+        >
+          {label}:
```

Comment:
So, this is failing the linter for a kind of dumb reason: Prettier and the linter are disagreeing with each other. That's fine and fixable (it's rare but it happens, and the underlying rules are usually useful).

But this is problematic for an unrelated reason: it's super-locale specific. Other languages may not use colons here. This should be a config option or something, but we aren't there yet with locale strings, so I think for now this should _just_ be `{label}`. If that means we need to use an `aria-describedby` prop we can, though I don't think it's needed. But I'll remove this colon for now.

Reply:
But then to screen readers, it'll read "Category books" or possibly even "Categorybooks", which seems sub-optimal. Is there a better way we can approach this?

Reply:
Using a different element and `aria-describedby` might be best. This approach is bad for localisation and will require awkward config. Described by might handle that for us; it's worth checking out to see if that's true.